index.bs

```diff
@@ -1058,8 +1062,25 @@ To <dfn>parse aggregatable values</dfn> given an [=ordered map=] |map|:
      1. If |value| is less than or equal to 0, return null.
 1. Return |values|.
 
-To <dfn noexport>parse trigger-registration JSON</dfn> given a [=string=]
-|json|, a [=site=] |destination|, and an [=origin=] |reportingOrigin|:
+To <dfn noexport>serialize a private state token</dfn> given a [=string=] |encodedBlindedPrivateStateToken|:
+1. If |encodedBlindedPrivateStateToken| is null, return null.
+1. Let |decoded| be the result of [=forgiving-base64 decoding=] |encodedBlindedPrivateStateToken|.
+1. If |decoded| is failure, return null.
+1. Let |tokens| be the result of finishing issuance of |decoded|.
+
+ISSUE: properly define the "finishing issuance" operation.
+
+1. If |tokens| is null, or has [=list/size=] not equal to 1, return null.
+1. Let |token| be |tokens|[0].
+1. Let |redeemedBytes| be the result of "beginning redemption" with |token|,
+    the empty [=byte sequence=] (data), and 0 (the null timestamp).
+
+ISSUE: properly define "begin redemption" operation. Consider running the algorithm at report sending time.
+
+1. Return the result of [=forgiving-base64 encoding=] |redeemedBytes|.
+
+To <dfn noexport>create an attribution trigger</dfn> given a [=string=]
+|json|, a [=site=] |destination|, an [=origin=] |reportingOrigin|, and a [=string=] |encodedPrivateStateToken|:
```

Comment:
I assume encoded indicates the token is base64 encoded? Can we drop it and simply say privateStateToken?

Reply:
sure